.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,6 +38,7 @@
 - This PR is related to issue:
 - Link to documentation pull request:
 - Link to cli pull request:
+- Link to client library pull request:
```

Comment:
Yeah this makes sense, and ideally we also add a pull request template to the client library, which links back.

This is not really related to this PR, so it should be a separate.